src/gromacs/nblib/molecules.h

```diff
@@ -96,19 +99,19 @@ class Molecule
                           const V&            charge,
                           ParticleType const& particleType) = delete;
 
-    // Add a particle to the molecule with implicit charge of 0
+    //! Add a particle to the molecule with implicit charge of 0
```

Comment:
Adding the ! causes doxygen errors. Run "make check-source" to see them.

Reply:
Don't we have the ! in many other places? And aren't comments without the ! treated as just internal comments that won't show up in Doxygen?

Reply:
I think the issue is related to the ancient version of doxygen we use not handling template parameters well. The options are doxygen comments or no ! as far as I can tell.

Reply:
That's an easy one, I'll take "no !", thank you.